src/03-modality-agnostic-files.md

```diff
@@ -54,6 +54,67 @@ Example:
+
+As for any BIDS dataset a `dataset_description.json` file MUST be found at the
+top level of the a derived dataset:
+`<dataset>/derivatives/<pipeline_name>/dataset_description.json`
+
+In addition to the keys for raw BIDS datasets,
+derived BIDS datasets include the following REQUIRED and RECOMMENDED
+`dataset_description.json` keys:
+
+| **Key name**   | **Description**                                                                                                                                                                    |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| GeneratedBy    | REQUIRED. List of [objects][object] with at least one element.                                                                                                                     |
+| SourceDatasets | RECOMMENDED. A list of [objects][object] specifying the locations and relevant attributes of all source datasets. Valid fields in each object include `URL`, `DOI`, and `Version`. |
+
+Each object in the `GeneratedBy` list includes the following REQUIRED, RECOMMENDED
+and OPTIONAL keys:
+
+| **Key name** | **Description**                                                                                                                                                                                              |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Name         | REQUIRED. Name of the pipeline or process that generated the outputs. Use `"Manual"` to indicate the derivatives were generated by hand, or adjusted manually after an initial run of an automated pipeline. |
+| Version      | RECOMMENDED. Version of the pipeline.                                                                                                                                                                        |
+| Description  | OPTIONAL. Plain-text description of the pipeline or process that generated the outputs. RECOMMENDED if `Name` is `"Manual"`.                                                                                 |
+| CodeURL      | OPTIONAL. URL where the code used to generate the derivatives may be found.                                                                                                                                  |
+| Container    | OPTIONAL. [Object][object] specifying the location and relevant attributes of software container image used to produce the derivative. Valid fields in this object include `Type`, `Tag` and `URI`.          |
+
+Example:
+
+```JSON
+{
+  "Name": "FMRIPREP Outputs",
+  "BIDSVersion": "1.4.0",
+  "GeneratedBy": [
+    {
+      "Name": "fmriprep",
+      "Version": "1.4.1",
```

Comment:
Updated to be consistent with

https://github.com/bids-standard/bids-specification/blob/20b22698c6886054bca4be790ae791e174eec1f9/src/02-common-principles.md#L208

Following up on https://github.com/bids-standard/bids-specification/pull/488#discussion_r433277552.